### PR TITLE
[wallet-ext] Add sentry logging to wallet

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by Jordan-Mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #8602** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/8602" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/8602?utm_source=stack-comment).

**Review by Jibz1 — APPROVED** [review]

File: apps/wallet/src/ui/app/SentryRpcClient.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Mysten Labs, Inc.
```
Comment (williamrobertson13) [inline apps/wallet/src/ui/app/SentryRpcClient.ts:1]: does it make sense to maintain separate but identical `SentryRpcClient` classes for the wallet and explorer? or might we have slightly different constraints based on the use case?

**Review by williamrobertson13 — APPROVED** [review]
one small comment but lgtm!

File: apps/wallet/src/ui/app/SentryRpcClient.ts (hunk 1 above)
Comment (Jordan-Mysten) [reply apps/wallet/src/ui/app/SentryRpcClient.ts:1]: Nope! It doesn't at all 😅 
We've been meaning to create a shared location for code like this for a while (`@mysten/core` is intended to be _roughly_ this) we just haven't integrated it with the projects yet.

**Pull request merged**